opal/mca/btl/ofi/btl_ofi_component.c

```diff
@@ -268,8 +259,12 @@ static mca_btl_base_module_t **mca_btl_ofi_component_init (int *num_btl_modules,
             break;
     }
 
-    /* Select the provider */
-    fabric_attr.prov_name = prov_include;
+    fabric_attr.prov_name = NULL;
+    /* Select the provider - sort of.  we just take first element in list for now */
+    if (NULL != *opal_common_ofi.prov_include) {
+        include_list = opal_argv_split(*opal_common_ofi.prov_include, ',');
+        fabric_attr.prov_name = include_list[0];
+    }
```

Comment:
This doesn't seem like the correct behavior...It ignores all providers except the first one and ignores the exclude list as well. Not saying it needs to be done in this patch, but can you make an issue so we can track this?

Reply:
@wckzhang, @hppritcha  ignoring all providers except the first one seems to match the original btl/ofi code and the pre-existing README, which says that btl/ofi will initialize a module with ONLY the first compatible info returned from OFI.  I might be reading it wrong, but it seems that currently there is no btl/ofi exclude list? Should there be? (Or William, are you saying that the btl/ofi should look a the mtl/ofi exclude list?)

Reply:
@hkuno I think the answer to your question is yes.  That's part of why I marked this as a WIP.  If we'd like to have the BTL work like the MTL in terms of include/exclude I'd be fine with adding that cap.  I'd do that as  a separate commit to this PR.

Anyone object to adding an ofi mtl like include/exclude behavior to the ofi btl?

Reply:
I think that's a good idea Jeff

Reply:
Thank you for both the current commit and for volunteering to do the new one!

Reply:
@jsquyres this PR already includes the aliasing.  See the common_ofi.c

Reply:
this is going to change if PR #7641 common ofi provider selection method is pulled in.